eternalcore/build.gradle.kts

```diff
@@ -87,10 +99,13 @@ tasks.withType<ShadowJar> {
     relocate("io.papermc.lib", "com.eternalcode.core.libs.io.papermc.lib")
     relocate("net.kyori", "com.eternalcode.core.libs.net.kyori")
 
-    relocate("fr.mrmicky.fastboard", "com.eternalcode.core.libs.fr.mrmicky.fastboard")
     relocate("dev.triumphteam.gui", "com.eternalcode.core.libs.dev.triumphteam.gui")
 
-    relocate("com.zaxxer", "com.eternalcode.core.libs.com.zaxxer")
     relocate("org.slf4j", "com.eternalcode.core.libs.org.slf4j")
     relocate("com.google.gson", "com.eternalcode.core.libs.com.google.gson")
+
+//    copy {
```

Comment:
Czy to w ogóle jest obecnie potrzebne? Wydaje mi się ze debug można wykonać przy pomocy run-paper?

Reply:
do wywalenia, ja sobie tego używałem lokalnie